### PR TITLE
doc: week 7 reviewer notes

### PR DESCRIPTION
**Pull request by jdevries3133**

### CODE REVIEW

File: reviewer-notes/WEEK_7.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Summary
+
+[Lesson page](https://learn.codethedream.org/ctd-node-express-class-lesson-9-using-query-parameters/)
+
+From the lesson page:
+
+> This is a difficult lesson, so take your time with it...
+
+This week, students finish the Store API assignment (04).
```
Comment (akosasante) [inline reviewer-notes/WEEK_7.md:9]: To clarify, this is the first/only week that they work in the `04-store-api` directory right? Or is there some work there from week 6?

File: reviewer-notes/WEEK_7.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Note that students' PRs will probably have the contents of this week and the
+previous week combined.
```
Comment (akosasante) [inline reviewer-notes/WEEK_7.md:12]: To clarify, this is the first/only week that they work in the `04-store-api` directory right? Or is there some work there from week 6?

**Comment by jdevries3133** [comment]
@akosasante PTAL – these notes were quite jank before. Certainly, I generally rely on a really great student submission to make these notes very good, so I was hoping to get something clear from a student and revise these notes, but either forgot or never got anything super clear to build from.

Despite my overall confusion about this week, I think these notes basically communicate what's going on!

**Comment by jdevries3133** [comment]
Closing; prefer https://github.com/Code-the-Dream-School/school-pages/pull/25

**Pull request closed without merging**